examples/tutorials/Part 12 - Train an Encrypted Neural Network on Encrypted Data.ipynb

```diff
@@ -183,7 +198,7 @@
     }
    ],
    "source": [
-    "model_enc.get().float_precision()"
+    "model.weight.child.child.get().float_precision()"
```

Comment:
Is there anything we can do to get the user to not call .child?

Reply:
Sure!

Reply:
Done